app/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.test.tsx

```diff
@@ -166,6 +166,32 @@ describe("FormAudience", () => {
     );
   });
 
+  it("requires positive numbers in numeric fields (EXP-956)", async () => {
+    const onSubmit = jest.fn();
+    const { container } = render(<Subject {...{ onSubmit }} />);
+    await waitFor(() => {
+      expect(screen.queryByTestId("FormAudience")).toBeInTheDocument();
+    });
+
+    for (const fieldName of [
+      "populationPercent",
+      "totalEnrolledClients",
+      "proposedEnrollment",
+      "proposedDuration",
+    ]) {
+      await act(async () => {
+        const field = screen.getByTestId(fieldName);
+        fireEvent.click(field);
+        fireEvent.change(field, { target: { value: "-123" } });
+        fireEvent.blur(field);
+      });
+
+      expect(
+        container.querySelector(`.invalid-feedback[data-for=${fieldName}`),
```

Comment:
My Spidey-senses are tingling here a bit.
Is this missing a closing `]` (for the `data-for=` attribute)?

Reply:
Hmm, apparently not? Because I copypasta'd that from another spot. But it really should have one, so let's see what fixing that does

Reply:
Well, now I want to know where we copy-pastad from so I can kvetch about that instance too.

Reply:
Filed as https://github.com/mozilla/experimenter/issues/4715